app/main/src/main.ts

```diff
@@ -305,40 +333,30 @@ if (!onlySingleInstance) {
 
     const autoUpdater = activateAutoUpdate({
       onUpdateAvailable: (info) => {
-        notifier.notify(
-          {
-            icon: notificationIcon,
-            title: "NEW UPDATE IS AVAILABLE",
-            message: `App version ${info.version} ready to be downloaded.`,
-            actions: ["View Released Notes"],
-            sound: true,
-            wait: true,
-          },
-          (err, response) => {
+        notify({
+          title: "NEW UPDATE IS AVAILABLE",
+          message: `App version ${info.version} ready to be downloaded.`,
+          actions: ["View Released Notes"],
+          callback: (err, response) => {
             if (!err) {
               shell.openExternal(RELEASED_NOTES_LINK);
             }
-          }
-        );
+          },
+        });
       },
       onUpdateDownloaded: (info) => {
-        notifier.notify(
-          {
-            icon: notificationIcon,
-            title: "READY TO BE INSTALLED",
-            message: "Update has been successfully downloaded.",
-            actions: ["Quit and Install", "Install it Later"],
-            sound: true,
-            wait: true,
-          },
-          (err, response) => {
+        notify({
+          title: "READY TO BE INSTALLED",
+          message: "Update has been successfully downloaded.",
+          actions: ["Quit and Install", "Install it Later"],
+          callback: (err, response) => {
             if (!err) {
               if (response === "quit and install") {
```

Comment:
For some reason this callback seems to be undefined. Though the update installs anyway.

So the icons show fine but something is odd with the response.

When you dismiss it it does seem to respond with dismissed. Ill need to check the events on windows and mac as well as any other callback logic functions as expected.